src/app/containers/PageViewAnalytics/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Page View Analytics Container should render amp page view analytics correctly 1`] = `null`;
```

Comment:
Should this be `null` are we handling `amp` in another issue?

Reply:
Yeah the idea was this remains as null untill the amp work is done

Reply:
Not very AMP-first 🙊